lib/index.js

```diff
@@ -60,6 +60,8 @@ export default function rehypePicture(options) {
       }
 
       const src = String(node.properties.src)
+      const width = node.properties?.width
+      const height = node.properties?.height
```

Comment:
```suggestion
      const width = node.properties.width
      const height = node.properties.height
```

The `?` is not needed.